DigitalLearningSolutions.Web/Controllers/Register/RegisterAdminController.cs

```diff
@@ -201,26 +215,35 @@ private void SetAdminRegistrationData(int centreId)
             TempData.Set(adminRegistrationData);
         }
 
+        private bool CheckEmailUnique(string email)
+        {
+            var adminUser = userDataService.GetAdminUserByEmailAddress(email);
+            return adminUser == null;
+        }
+
+        private bool CheckEmailMatchesCentre(string email, int centreId)
+        {
+            var autoRegisterManagerEmail =
+                centresDataService.GetCentreAutoRegisterValues(centreId).autoRegisterManagerEmail;
+            return email.Equals(autoRegisterManagerEmail);
+        }

```

Comment:
Minor naming thing. I'd call these something that signifies they return bool a bit more clearly e.g. IsEmailUnique & DoesEmailMatchCentre